python/src/main/api.py

```diff
@@ -51,9 +63,35 @@ def get(self, email):
             return jsonify({'person': create_person(person).serialize()})
 
 
-# @api.route('/api/v1/items/<string:email>')
-# class Item(Resource):
-#     def put(self, ):
+@api.route('/api/v1/items/<string:person_email>/<string:name>/<string:price>/<int:priority>/<string:hyperlink>')
+class Items(Resource):
+     def put(self, person_email, name, price, priority, hyperlink):
+        conn = sqlite3.connect("person_and_item.db")
+        cursor = conn.cursor()
+        db = DB(cursor, conn)
+        item = Item(person_email, name, price, priority, hyperlink)
+        db.insert_item(item)
+        logging.info("Put item '%s'" % name)
+        conn.close()
+        return 200
+
+@api.route('/api/v1/items/<string:person_email>/<string:name>')
+class Itemz(Resource):
+     def delete(self, person_email, name):
```

Comment:
this method can be done in the same class as the above put!!

Reply:
Made new class because I had made a new api.route, didn't know you could have same class across more than one api.route

Reply:
Again, google it and see if you can or not

Reply:
Even so, Itemz is not a good name!